modules/foliate/testbeds/default.nix

```
@@ -0,0 +1,13 @@
+{ lib, pkgs, ... }:
+
+let
+  package = pkgs.foliate;
+in
+{
+  stylix.testbed.ui.application = {
+    name = "com.github.johnfactotum.Foliate";
+    inherit package;
+  };
+  home-manager.sharedModules = lib.singleton { home.packages = [ package ]; };
+
```

very small formatting nit
```suggestion

  home-manager.sharedModules = lib.singleton { home.packages = [ package ]; };
```